cmd/download.go

```diff
@@ -59,21 +59,21 @@ func runDownload(cfg config.Config, flags *pflag.FlagSet, args []string) error {
 	if err != nil {
 		return err
 	}
-	exercise, err := flags.GetString("exercise")
+	slug, err := flags.GetString("exercise")
 	if err != nil {
 		return err
 	}
-	if uuid == "" && exercise == "" {
+	if uuid == "" && slug == "" {
 		return errors.New("need an --exercise name or a solution --uuid")
 	}
 
-	var slug string
+	var param string
 	if uuid == "" {
```

Comment:
Yon can simplify this a bit
```
param := "latest"
if uuid != "" {
 param = uuid
}
```

Reply:
Thanks! Yeah, that's much better. Updated.